backend/account/subscription_loader.py

```diff
@@ -75,3 +77,32 @@ def load_plugins() -> list[Any]:
         logger.info("No subscription plugins found.")
 
     return subscription_plugins
+
+
+def etl_prerun_check(org_id: str) -> bool:
+    """Method to check subscription status before ETL runs.
+
+    Args:
+        org_id: The ID of the organization.
+
+    Returns:
+        A boolean indicating whether the pre-run check passed or not.
+    """
+    try:
+        from pluggable_apps.subscription.subscription_helper import SubscriptionHelper
```

Comment:
Can we avoid this import by getting the module from subscription_plugins list iteself?